src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -1087,8 +1080,8 @@ public async Task DisableTwoFactorProviderAsync(Organization organization, TwoFa
             throw new BadRequestException("Organization must have at least one confirmed owner.");
         }
 
-        var orgUsers = new List<OrganizationUser>();
-        var limitedCollectionOrgUsers = new List<(OrganizationUser, IEnumerable<CollectionAccessSelection>)>();
+        var orgUsersWithoutCollections = new List<OrganizationUser>();
+        var orgUsersWithCollections = new List<(OrganizationUser, IEnumerable<CollectionAccessSelection>)>();
```

Comment:
This fixes a latent bug - new orgUsers were sorted into these 2 lists based on collection access, but we only returned the first list as successfully invited users (`orgUsers`, being those without collection access). This was causing tests to fail, because if you invite 1 orgUser with collection access, it wasn't returning it in the success result.

I've renamed the lists to reflect how they are actually used, and updated the code to return all orgUsers.